src/styles/base/_tables.scss

```diff
@@ -16,8 +16,13 @@ td {
 	}
 }
 
-tr:last-child td,
-tr:last-child th {
+tbody:last-child > tr:last-child td {
```

Comment:
Added a test case for with/without tbody/thead/tfoot. I'm confident this isn't going to break things!  Tested in a few (modern) browsers and all looks good.